# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1456777317 18000
#      Mon Feb 29 15:21:57 2016 -0500
# Node ID 1233bf2a03a593dcdcc9ca5dfd6b6c8e5172738c
# Parent  e5fd5bc5fd902ca5b695ce86875be172090a01f4
* plot.txi: Fix statement about changing graphics toolkit property for figure.

diff --git a/doc/interpreter/plot.txi b/doc/interpreter/plot.txi
--- a/doc/interpreter/plot.txi
+++ b/doc/interpreter/plot.txi
@@ -37,18 +37,17 @@ capability is provided by access to Open
 is used is controlled by the @code{graphics_toolkit} function.
 @xref{Graphics Toolkits}.
 
 The function call @code{graphics_toolkit ("qt")} selects the
 Qt/OpenGL system, @code{graphics_toolkit ("fltk")} selects the
 FLTK/OpenGL system, and @code{graphics_toolkit ("gnuplot")} selects the
 gnuplot system.  The three systems may be used selectively through the use
 of the @code{graphics_toolkit} property of the graphics handle for each
-figure.  The toolkit used for a given figure may not be changed once the
-figure is initialized.  This is explained in @ref{Graphics Data Structures}.
+figure.  This is explained in @ref{Graphics Data Structures}.
 @strong{Caution:} The OpenGL-based toolkits use single precision
 variables internally which limits the maximum value that can be
 displayed to approximately @math{10^{38}}.  If your data contains larger
 values you must use the gnuplot toolkit which supports values up to
 @math{10^{308}}.
 
 @node High-Level Plotting
 @section High-Level Plotting
